crates/load-balancer/src/configuration.rs

```diff
@@ -3,12 +3,13 @@ use std::collections::HashMap;
 
 #[derive(Debug, Deserialize)]
 pub struct Config {
+    pub services: HashMap<String, String>,
     pub backends: Vec<BackendConfig>,
 }
 
 #[derive(Debug, Deserialize)]
 pub struct BackendConfig {
-    pub prefix: String,
```

Comment:
Can you add validation for referential consistency?

1. All service mentioned in the Backend must be defined in the services field.
2. All service must have at least one backend.